src/content/content.service.ts

```
@@ -0,0 +1,336 @@
+import * as crypto from 'crypto';
```

ESLint

[error] Replace `'crypto'` with `"crypto"`

(prettier/prettier)